### PR TITLE
Add progress bar + Nikole's prototype updates

### PR DESCRIPTION
**Pull request by copecopecope**
Adding progress bar and Nikole's prototype updates since July which should wrap up the statement selection part of the flow!

https://user-images.githubusercontent.com/2431866/135346792-5de66c1b-ba1a-4a3e-b171-d76a128e8d1e.mov

### CODE REVIEW

File: draftivist_ui/src/components/campaign/Campaign.tsx [hunk 1]
```diff
@@ -17,6 +17,30 @@ export default function() : BaseComponent<Attrs> {
 
     let campaign: Campaign = new Campaign(0, "", "")
 
+    function getTranslateX(page: string, issue_page?: string): string {
```
Comment (NonsenseSynapse) [inline draftivist_ui/src/components/campaign/Campaign.tsx:20]: Could you provide a more robust docstring explaining what this function does?

File: draftivist_ui/src/components/campaign/CampaignIssue.tsx [hunk 2]
```diff
@@ -90,7 +90,7 @@ export default function (): BaseComponent<Attrs> {
                         Next
                 </a>
                 <Dialog id="source">
-                    <span>These statements were provided by the <strong>{campaign.organizer}</strong>.</span>
+                    <span>These statements were written and verified by the {campaign.organizer}.</span>
```
Comment (NonsenseSynapse) [inline draftivist_ui/src/components/campaign/CampaignIssue.tsx:93]: I wonder if "the" before `{campaign.organizer}` should be removed. Feels like that could cause issues if an organization's name already begins with "the"

**Comment by NonsenseSynapse** [comment]
Looks really good. I noticed that the images seem to not be displaying properly on staging. Any support I can provide around getting those working?

**Comment by copecopecope** [comment]
> Looks really good. I noticed that the images seem to not be displaying properly on staging. Any support I can provide around getting those working?

Sorry for taking forever to get back to you. I think all we need to do is update `STAGING_PATH` env var with an extra `/`. Adding that to the pr

**Pull request merged**